python/quantum-pecos/src/pecos/qeclib/steane/steane_class.py

```diff
@@ -400,7 +402,7 @@ def mz(self, log: Bit | None = None):
         """Logical destructive measurement of the logical Z operator."""
         return self.m("Z", log=log)
 
-    def qec(self, flag_bit: Bit | None = None):
+    def qec(self, flag: Bit | None = None):
```

Comment:
I also renamed `flag_bit` --> `flag` in this PR, to agree with conventions elsewhere in the repo.